src/config.ts

```diff
@@ -58,6 +60,56 @@ enum EnvVarType {
     JsonStringList,
 }
 
+export interface ApiKeyStructure {
+    [key: string]: {
+        label: string;
+        rfqt: boolean;
+        rfqm: boolean;
+        plp: boolean;
+    };
+}
+
+export const getApiKeyWhitelistWithFallback = (
```

Comment:
I personally think it's a bit overkill to support both old and new, we could just deploy with a new env var and just support new

Reply:
not blocking, just preference

Reply:
So you would perform the rollout by:
1. Copying API keys to new format
2. Deploying new code
3. Removing old API keys?

My strategy is to:
1. Deploy the new code, make sure it works well
2. Update environment variables (move old keys to new keys)

Reply:
Yea my pref is to just add copy the API keys to the new format, delete the old env var, and deploy (all in one main-infra PR), test on staging first.

Reply:
If the new key is there it'll use the new key, so it either works or it doesn't, if it doesn't we roll back.